game/app.py

```diff
@@ -38,7 +38,8 @@ def __call__(self, environ, start_response):
     "SESSION_COOKIE_NAME": "flask-session-id",
     "SESSION_COOKIE_HTTPONLY": True,
     "SESSION_COOKIE_SECURE": False,  # should be True in case of HTTPS usage (production)
-    "SESSION_COOKIE_SAMESITE": None  # should be 'None' in case of HTTPS usage (production)
+    "SESSION_COOKIE_SAMESITE": None,  # should be 'None' in case of HTTPS usage (production)
+    "APPEND_TIMEZONE" : False # Must be set to true if using Blackboard Learn
```

Comment:
I think this is not necessary.
https://www.imsglobal.org/spec/lti-ags/v2p0#example-of-valid-timestamps here is examples of valid timestamps:
```
"timestamp": "2017-04-16T18:54:36.736+00:00"
"timestamp": "2017-04-16T18:54:36.736Z"
"timestamp": "2017-04-16T18:54:36.736+00"
```
and looks like `isoformat()` doesn't fit this formats. So it is just coincidence that it works correct for Canvas and etc. I think we should always add `Z` without settings.